api/python/cellxgene_census/src/cellxgene_census/experimental/util/_csr_iter.py

```diff
@@ -94,35 +114,44 @@ def X_sparse_iter(
     if use_eager_fetch:
         table_reader = (t for t in _EagerIterator(table_reader, query._threadpool))
 
-    # lazy reindex of obs coordinates. Yields coords and (data, i, j) as numpy ndarrays
+    # Lazy reindexing of soma_dim_0 and soma_dim_1 to obs_coords and var_coords
+    # positions (except one or the other if not reindex_sparse_axis).
+    # Yields coords and COO (data, i, j) as numpy ndarrays
```

Comment:
nit: I found this comment is slightly confusing - it yields `((obs_coords, var_coords), (data, i, j))` so perhaps be specific?